tests/sources/test_onedrive.py

```diff
@@ -60,54 +60,54 @@ def test_get_configuration():
 async def test_validate_configuration_with_invalid_dependency_fields_raises_error(
     extras,
 ):
-    source = create_source(OneDriveDataSource, **extras)
-
-    with pytest.raises(ConfigurableFieldValueError):
-        await source.validate_config()
+    async with create_source(OneDriveDataSource, **extras) as source:
```

Comment:
Changes from this file seems not related to e2e, and I see them also changed in PR #1414